docs: clarify tracing name parameter semantics (#28759)
microsoft/playwright: docs/src/api/class-tracing.md

* since: v1.12
- `name` <[string]>

If specified, intermediate trace files are going to be saved into the files with the
given name prefix inside the [`option: tracesDir`] folder specified in [`method: BrowserType.launch`].

> If you are looking for an option to specify ouput zip file name, you need to pass `path` option to

Reviewer: ```suggestion
To specify the final trace zip file name, you need to pass `path` option to
```

* since: v1.32
- `name` <[string]>

If specified, intermediate trace files are going to be saved into the files with the
given name prefix inside the [`option: tracesDir`] folder specified in [`method: BrowserType.launch`].

> If you are looking for an option to specify ouput zip file name, you need to pass `path` option to

Reviewer: ```suggestion
To specify the final trace zip file name, you need to pass `path` option to
```
